README.md

```diff
@@ -1 +1,81 @@
-# angular-parsers-formatters-readme
\ No newline at end of file
+# $parsers and $formatters
+
+## Overview
+
+Parsers and formatters allow us to have different values in our view and our model for the same item. Don't be confused, they're really simple!
+
+## Objectives
+
+- Describe $parsers and $formatters
+- Use a $parser to change Model to View data
+- Use a $formatter to change View to Model data
+
+## $parsers and $formatters
+
```

Comment:
Add a longer description (a paragraph or two) about the problem that parsers and formatters solve. Why is this important?